lib/services/events/analyzers/google-analytics.js

```diff
@@ -18,6 +18,7 @@ module.exports = options => {
 
 Analyzer.prototype = {
 	prepare(payload) {
+		payload = payload.attributes;
```

Comment:
So in our test server implementation there is a `deformatter` to make it easier to work with the data you care about. I've been considering removing these and I haven't decided yet, but I think this example might have been the turning point.

https://github.com/oddnetworks/oddworks/blob/master/test/support/test-config.js#L92